src/base/static/components/molecules/user-menu.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { translate } from "react-i18next";
+import { Link } from "../atoms/typography";
+import classNames from "classnames";
+import LegacyUtil from "../../js/utils.js";
+
+class UserMenu extends React.Component {
+  state = {
+    isMenuOpen: false,
+  };
+
+  toggleMenu = () => {
+    this.setState(prevState => {
+      return {
+        ...prevState,
+        isMenuOpen: !prevState.isMenuOpen,
+      };
+    });
+  };
+
+  render() {
+    const {
+      slug: datasetSlug,
+      owner: datasetOwner,
+    } = this.props.datasetDownloadConfig;
+    const isAdmin = LegacyUtil.getAdminStatus(datasetSlug);
+    if (this.props.currentUser) {
+      // If user is logged in
+      return (
+        <nav className="user-menu authed" role="article">
+          <a href="#" id="sign-in-btn" onClick={this.toggleMenu}>
+            <img
+              className="avatar header-avatar"
+              src={this.props.currentUser.avatar_url}
+            />
+          </a>
+          <ul
+            className={classNames("menu", "sign-in-menu", {
+              "is-exposed": this.state.isMenuOpen,
+            })}
+          >
+            {isAdmin && (
+              <li style={{ paddingBottom: "16px" }}>
+                <Link
+                  href={`${
+                    this.props.apiRoot
+                  }${datasetOwner}/datasets/${datasetSlug}/mapseed-places.csv?format=csv&include_private=true&page_size=10000`}
+                  classes={".btn-secondary"}
+                  style={{ width: "100%", textDecoration: "none" }}
```

Comment:
Ideally we shouldn't have to pass through legacy CSS classes like `btn-secondary`. The `Link` component should have data-driven styles in it that replicate the styling in the legacy scss files.

I think if you pass a `color="secondary"` prop to a `Link` it will adopt the style of the Sign-In button.